### PR TITLE
[BZ#2102680] Address load balancer service creation limitation when the IP is specified with the in-tree cloud provider (and OSP) 

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s): 4.10 and 4.11. 4.12 needs a separate PR.

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2102680

Link to docs preview: https://51467--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-custom.html#installation-osp-cloud-provider-lb-fip_installing-openstack-installer-custom

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by maxwelldb** [comment]
@eurijon Is this what you had in mind, here?

**Comment by kalexand-rh** [comment]
@eurijon, will you PTAL?

**Comment by eurijon** [comment]
Oh sorry, I had a review in place long time ago but didn't push the "Review changes" button... /o\
So the change looks good to me, but as it's a limitation for the in-tree cloud provider (only used for OCP 4.11-), I would add this note to OCP 4.11- versions, and not in 4.12.

/cc @MaysaMacedo what do you think?

**Comment by MaysaMacedo** [comment]
There is also some extra configurations when using external cloud provided (supported on 4.12):
```
You can pre-create a floating IP and pass the address of it in the service
specification if using the external cloud provider
```
But mentioning the in-tree cloud provider in the 4.12 docs might be misleading as it stopped being supported in 4.12.

File: modules/installation-osp-cloud-provider-lb-fip.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-installer-user.adoc
+
+:_content-type: CONCEPT
+[id="installation-osp-cloud-provider-lb-fip_{context}"]
+= External load balancers that use pre-defined floating IP addresses
+
+Commonly, {rh-openstack-first} deployments disallow non-administrator users from creating specific floating IP addresses. If such a policy is in place and you use a floating IP address in your service specification, the cloud provider will fail to handle IP address assignment to load balancers.
+
+If you use an external cloud providers, you can avoid this problem by pre-creating a floating IP address and specifying it in your service specification. The in-tree cloud provider does not support this method.
```
Comment (maxwelldb) [inline modules/installation-osp-cloud-provider-lb-fip.adoc:13]:
```suggestion
If you use an external cloud provider, you can avoid this problem by pre-creating a floating IP address and specifying it in your service specification. The in-tree cloud provider does not support this method. 
```

**Comment by bergerhoffer** [comment]
@maxwelldb A few things with this PR. It targets main, so it should have gone into 4.12 too. If it wasn't for 4.12, then it should've gone right into 4.11 / 4.11 directly. Also, it has 2 commits.

Right now main and 4.12 are out of sync. Can you please fix it so that main and 4.12 are in sync again?

**Pull request merged**